odbc-api/src/handles/connection.rs

```diff
@@ -186,4 +186,25 @@ impl<'c> Connection<'c> {
                 .into_result(self)
         }
     }
+
+    /// Get the name of the database management system used by the connection.
+    pub fn database_management_system_name(
+        &self,
+        name: &mut OutputStringBuffer,
```

Comment:
Alright sounds good :+1: I'll make that change.

I had actually started down that path but then thought we might want to reuse `OutputStringBuffer` here, but I agree it would be better to deal with this internally.